evap/evaluation/models.py

```diff
@@ -468,6 +468,30 @@ class State(models.IntegerChoices):
         verbose_name=_("wait for grade upload before publishing"), default=True
     )
 
+    @property
+    def has_exam(self):
```

Comment:
@richardebeling Do we prefer this over `has_exam_evaluation`? (same for `earliest_possible...`); for creation, we also didn't call it `create_exam`. I think I prefer `exam_evaluation`

Reply:
I am ok with using has_exam_evaluation. I like it more than exam_evaluation as you do not get the evaluation object

Reply:
Ah yeah, sorry, by "I think I prefer `exam_evaluation`, I meant that I prefer to use "exam_evaluation" over just "exam" for everything, but still with "has_" and "create_".